### PR TITLE
Fontawesome 5 upgrade 14460

### PR DESCRIPTION
**Pull request by aubreyarcangel**
This PR is an upgrade from Font Awesome 4 icon library to Font Awesome 5. 

The icon library upgrade is reflected sitewide on the homepage, hub pages, modals, and other areas. 

Icon updates on the homepage alert banner, hub links containers:
screen shot 2019-01-11 at 3 42 48 pm
homepage benefits rows and homepage buttons
screen shot 2019-01-11 at 3 44 20 pm

screen shot 2019-01-11 at 3 48 39 pm

Before:
screen shot 2019-01-11 at 3 52 39 pm

After:
screen shot 2019-01-11 at 3 54 24 pm

Veteran Crisis Modal before:
screen shot 2019-01-11 at 4 19 57 pm

Veteran Crisis Modal after: 
screen shot 2019-01-11 at 4 22 18 pm

This PR needs to be pulled alongside this corresponding vets-website PR: https://github.com/department-of-veterans-affairs/vets-website/pull/9369

### CODE REVIEW

**Comment by ncksllvn** [comment]
In our React code, I see a FA reference in [SortableTable](https://github.com/department-of-veterans-affairs/design-system/blob/master/src/components/SortableTable/SortableTable.jsx#L33) that should be updated.

A similar reference in [AdditionalInfo](https://github.com/department-of-veterans-affairs/design-system/blob/master/src/components/AdditionalInfo/AdditionalInfo.jsx#L22).

I think this [footer file](https://github.com/department-of-veterans-affairs/design-system/blob/master/src/components/footer/footer.njk#L74) could use an update as well.

**Comment by ncksllvn** [comment]
If we can make the updates listed above, I think we can merge this to `master`, then publish this to NPM as a new major version. Then, we can update your PR in `vets-website`, https://github.com/department-of-veterans-affairs/vets-website/pull/9369, to use the newly-published version of Formation. Then, our design team can help us test using the Heroku review instance.

@aubreyarcangel For context, we use [semantic versioning](https://semver.org/) for publishing to NPM. This would be a major change, and to reflect that, we need to update these two files to version `2.0.0` -

- https://github.com/department-of-veterans-affairs/design-system/blob/master/package.json#L3
- https://github.com/department-of-veterans-affairs/design-system/blob/master/module-readme.md

@jbalboni @rianfowler  Does this make sense to you as well? Or maybe publish as an alpha version first?

**Comment by jbalboni** [comment]
I think an alpha version would be good, if you think this is going to take a while to test. 

We're probably going to substantially reorganize this repo this week, FYI. So we may need to coordinate a little bit on the order of changes.

**Comment by ncksllvn** [comment]
Sounds good, @aubreyarcangel and I are planning to publish version `2.0.0-alpha` later today

**Comment by jbalboni** [comment]
Also, we should avoiding merging this while testing on the alpha version, so we don't block other changes.

**Comment by ncksllvn** [comment]
https://github.com/department-of-veterans-affairs/design-system/pull/220/commits/e44f76b2f7030dbf4f0dda88e60973deb11d922d has been published into NPM as @department-of-veterans-affairs/formation@2.0.0-fa

File: src/components/SortableTable/SortableTable.jsx [hunk 1]
```diff
@@ -31,8 +31,8 @@ class SortableTable extends React.Component {
     if (this.props.currentSort.value === field.value) {
       const iconClass = classNames({
         fa: true,
-        'fa-caret-down': this.props.currentSort.order === 'DESC',
```
Comment (ncksllvn) [inline src/components/SortableTable/SortableTable.jsx]: @aubreyarcangel Your update here works, but you could have also just changed `fa: true` to `fas: true`. The [`classNames` module](https://www.npmjs.com/package/classnames) as seen here is a widely-used utility function for a constructing an HTML class out of a list of conditions.

**Pull request merged**